src/include/catalog.h

```diff
@@ -0,0 +1,24 @@
+#include <vector>
+#include "table.h"
+#include <string>
+
+namespace emerald {
+    class Catalog {
+        private:
+          std::vector<Table*> TableList;
```

Comment:
The two map instance variables are missing, please add them in.

Reply:
What is the map instance?